public/style.css

```diff
@@ -19,3 +19,34 @@
 .ol-zoom-extent button{ 
     background-image: url("extentButton.png");
 }
+
+.button {
+    height: 40px;
+    width: 40px;
+    border: 2px solid blue;
+    display: inline-block;
+    background-size:100% 100%;
+    cursor: pointer;
+}
+.firstFrame {
+    background-image: url("first.png");
```

Comment:
My first thought would have been to create an `img` tag nested within the button...I like this.